pkg/controller/kogitosupportingservice/kogitosupportingservice_controller.go

```diff
@@ -229,4 +229,5 @@ var kogitoSupportingServices = map[v1alpha1.ServiceType]SupportingServiceResourc
 	v1alpha1.Explainablity: &ExplainabilitySupportingServiceResource{},
 	v1alpha1.TrustyAI:      &TrustyAISupportingServiceResource{},
 	v1alpha1.TrustyUI:      &TrustyUISupportingServiceResource{},
+	v1alpha1.TaskConsole:   &TaskConsoleSupportingServiceResource{},
```

Comment:
I think we should put them in alphabetical order, will be easier to find them :)

Reply:
I guess the alphabetical order would go also for other files where the different supporting services are configured/listed

Reply:
done